packages/dds/map/src/localValues.ts

```diff
@@ -179,25 +179,32 @@ export class LocalValueMaker {
     /**
      * Create a new local value from an incoming serialized value.
      * @param serializable - The serializable value to make local
-     * @param emitter - The value op emitter, if the serializable is a value type
      */
-    public fromSerializable(serializable: ISerializableValue, emitter?: IValueOpEmitter): ILocalValue {
-        if (serializable.type === ValueType[ValueType.Plain] || serializable.type === ValueType[ValueType.Shared]) {
-            // Migrate from old shared value to handles
-            if (serializable.type === ValueType[ValueType.Shared]) {
-                serializable.type = ValueType[ValueType.Plain];
-                const handle: ISerializedHandle = {
-                    type: "__fluid_handle__",
-                    url: serializable.value as string,
-                };
-                serializable.value = handle;
-            }
+    public fromSerializable(serializable: ISerializableValue): ILocalValue {
+        // Migrate from old shared value to handles
```

Comment:
Is this for backcompat?

Reply:
looks like it.    Conversion to handles from older format.